tf2_ros_py/test/test_buffer_client.py

```diff
@@ -192,7 +192,7 @@ def test_lookup_transform_true(self):
             self.node, 'lookup_transform', check_frequency=10.0, timeout_padding=0.0)
 
         result = buffer_client.lookup_transform(
-            'foo', 'bar', rclpy.time.Time(), rclpy.duration.Duration(seconds=5.0))
+            'foo', 'bar', rclpy.time.Time().to_msg(), rclpy.duration.Duration(seconds=5.0))
```

Comment:
Why is this change (and the similar ones below) needed in this port, but not in the original?

Reply:
This comment is still open (and there are a couple more instances of it below).

Reply:
That's just the way it works now.  The original didn't work.  Resolving this conversation.